tests/integration/conftest.py

```diff
@@ -40,13 +35,26 @@ def observer_id(request):
 
 
 @pytest.fixture(scope="session")
-def connection(host):
-    return _sdk.from_local_account(host, os.environ["C42_USER"], os.environ["C42_PW"])
+def connection():
+    host = os.environ.get("C42_HOST") or "127.0.0.1:4200"
+    user = os.environ.get("C42_USER") or "test.user@example.com"
+    pw = os.environ.get("C42_PW") or "password"
+    return _get_sdk(host, user, pw)
+
+
+def _get_sdk(host, user, pw):
+    try:
+        return _sdk.from_local_account(host, user, pw)
+    except Exception as err:
+        pytest.exit(
```

Comment:
`echo $?` seems to prove this works, but is kind of odd since the tests show as green

Reply:
Are you running all tests or just integration when it shows green? For me running `pytest -m integration` highlights yellow since it failed before anything actually ran.

Reply:
I was running `pytest tests/test_users/py`.
I see the yellow when running all the tests, but I think that is because of other warnings

Reply:
Or perhaps some race condition. It seems to work though! In both cases